workflow/react-project/src/components/common/PasswordInput/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import { faEyeSlash } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import './style.less';
+
+const PasswordInput = ({ title, errorsMessage, ...inputProps }) => {
+  const [isPasswordVisible, setPasswordVisible] = useState(false);
+
+  const onPasswordIconClick = () => setPasswordVisible(!isPasswordVisible);
+
+  return (
+    <div className="authentication-password-input-wrapper">
```

Comment:
styles have to be written in the way it can be added to any place 
in common components can't be styles specified to some modal/page/etc. - 
`authentication-password-input-wrapper` - remove it or if you need wrapper, name it `input-wrapper`